src/main/java/softdesign/map/Map.java

```diff
@@ -1,11 +1,15 @@
 package main.java.softdesign.map;
 
+import javax.vecmath.Point3d;
+
 public interface Map {
 
 	enum Tile {
 		EMPTY, COVERED, WALL
 	}
 
+	Tile getTile(Point3d coord);
```

Comment:
Can you rename this to coordinate? Rest of the code also uses coordinate and it is considered bad practice to use non-standard abbreviations. (Simbad is not a good example to follow)